guide/coaching.md

```diff
@@ -8,24 +8,28 @@ permalink: /guide/coaching/
 ---
 <h1>Becoming a Coach</h1>
 
-<p>You heard about Rails Girls Summer of Code and want to be a coach for a team in your city? Here you’ll find all you need to know about what coaching actually means.</p>
+<p>You heard about Rails Girls Summer of Code and want to be a coach for a team? Here you’ll find all you need to know about what coaching actually means.</p>
 
 <ul>
 <li><a href="#intro">What it takes to be a coach</a></li>
 <li><a href="#why">Why should I be a coach?</a></li>
-<li><a href="#what">What does coaching require?</a></li>
+<li><a href="#remote">Can I coach remotely?</a></li>
+<li><a href="#what">What will I be doing as a coach?</a></li>
+<li><a href="#expertise">Should I have a good knowledge of a project chosen by my team?</a></li>
 <li><a href="#time">How much time does coaching take?</a></li>
 <li><a href="#register">Ways to find a team and register as a coach</a></li>
 </ul>
 
 <h3 id="intro">What it takes to be a coach</h3>
 
 * Patience and an open mind
-* Some experience in coaching (Rails Girls or other coding workshops)
+* Some experience in coaching
 * Time for coaching during office hours
 
 
-<p>First of all, coaches are patient, tolerant and open people! Ideally, they have some experience in coaching already - either from a Rails Girls event or something similar like a local workshop.</p>
+<p>First of all, coaches are patient, tolerant and open people! Our students are people with various coding levels (everything from novice to medium), as well as with diverse backgrounds, mentalities, mindsets, tempers and motivation. We want this to be considered and respected.</p>
```

Comment:
change “as well as with diverse backgrounds, mentalities, mindsets, tempers and motivation” to
“as well as diverse backgrounds, mentalities and mindsets”. (I don't know if motivation would really be a factor to consider here, and “tempers” just sounds weird).